### PR TITLE
Breadcrumbs IA and content updates

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/breadcrumbs.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Breadcrumbs are an inline visual representation of page relationships and hierarchy. They provide a user with visual reinforcement of the current location within the application hierarchy, and allow for quick movement up the hierarchy.
+  Breadcrumbs are a linear navigation scheme that shows the users' current location in a website or application. A user can click  breadcrumb elements to navigate up the site hierarchy to previously visited pages.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: " users' "

Comment (palindromebart) [reply _component_design/breadcrumbs.md]: BCs are a hierarchical representation of the users' current location. They are critical for wayfinding and navigation up the hierarchical chain.

File: _component_design/breadcrumbs.md [hunk 2]
```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
 <section class="static-section" markdown="1">
 
-Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page hierarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second level of nesting and deeper within a module.
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use breadcrumbs when you want to improve web application way-finding and navigation. Breadcrumbs help the user understand their location within an application, and provide an easy way for the user to navigate out of their current view.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: @araiford 
Should we give guidance on when NOT to use breadcrumbs?

File: _component_design/breadcrumbs.md (hunk 1 above)
Comment (chri2547) [inline _component_design/breadcrumbs.md]: breadcrumbs show current location...a user can navigate up the hierarchical chain.

Comment (palindromebart) [inline _component_design/breadcrumbs.md]: BC is a page navigation component that shows the users' current location and helps with wayfinding and navigating up the hierarchical chain.

Comment (palindromebart) [inline _component_design/breadcrumbs.md]: BC is a page navigation component that shows the users' current location and helps with wayfinding and navigating up the hierarchical chain.

File: _component_design/breadcrumbs.md (hunk 2 above)
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: nix 2nd sentence.

File: _component_design/breadcrumbs.md [hunk 3]
```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
-### When to use
+## Best practices
 
-When using breadcrumbs, keep the following in mind:
+Consider the following best practices when you construct breadcrumbs:
 
-- Keep the breadcrumbs brief and as minimal as possible.
-- Ensure the breadcrumbs go on the top of the page, above the header, and only show up on the second level of your app and deeper.
+- Label the breadcrumb item to match the page title to which it navigates.
+- If the page title is too long, use a shortened variation of the page title. For example, instead of `Billing overview`, use `Overview`. Shorter labels conserve horizontal space.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: or use truncation (xref to truncation page in text conventions)

File: _component_design/breadcrumbs.md [hunk 4]
```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
+- Place the breadcrumbs at the top of the page, above the header.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: move to top of list

File: _component_design/breadcrumbs.md [hunk 5]
```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
+- Only use breadcrumbs on the second level and deeper of a multi-layered application. Do use breadcrumbs on the first layer of a multi-layered application.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: make this the second item in list

File: _component_design/breadcrumbs.md [hunk 6]
```diff
@@ -18,30 +18,50 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
+
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: add guideline about how many crumbs to show

Comment (palindromebart) [reply _component_design/breadcrumbs.md]: Only show a max of four crumbs. If your application is more that 4 layers deep, see "too many" section below for how to truncate breadcrumbs.

Comment (palindromebart) [inline _component_design/breadcrumbs.md]: always show the top hierarchical level as the first bread crumb. 
Always show current location as the last crumb.
Always show the  one level above current location for the middle crumb.

___
Top > [...] > 2 levels above Current > Level above Current > Current 

Need a visual for this^

File: _component_design/breadcrumbs.md [hunk 7]
```diff
@@ -51,65 +71,112 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
...
-### Naming and Abbreviations
+### Spacing
+
+Use these spacing guidelines when you construct breadcrumbs.
+
+**OLD CONTENT - KEEP FOR NOW**
 
-- Ensure the items in the breadcrumbs have the same name as the page it navigates to. If the page title is longer than necessary, use shortened variations of the title. For example, instead of `Billing overview`, use `Overview`.
+- **Placement:** Keep breadcrumbs above the major heading of the page, 8px from the eyebrow. The container for breadcrumbs will always live on the top left corner of the page when it is present. Make sure the breadcrumb container is 16px above the content on the top of the page.
+- **Container & Spacing:** The breadcrumbs container is 28px tall and full-page wide if it needs to be. We recommend shortening measures below. Breadcrumbs are in all caps with arrow icons in `gray 600` in-between.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: probably can strike this content

File: _component_design/breadcrumbs.md [hunk 8]
```diff
@@ -51,65 +71,112 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
...
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Too many items
+## States
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: rearrange content into "too many" and "too long"

File: _component_design/breadcrumbs.md (hunk 1 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 2 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 3 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 4 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 5 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 6 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 2 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Will ask Drew on 1/7/2019.

Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Added that bit, done.

File: _component_design/breadcrumbs.md (hunk 6 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: I'm not entirely sure what you mean by your most recent comment. I came up with "Although breadcrumbs do not appear on the top-level page, ensure that the spacing in the header is consistent with child pages where breadcrumbs do appear."

Am I understanding your intent correctly?

File: _component_design/breadcrumbs.md [hunk 9]
```diff
@@ -18,30 +18,84 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
+## Best practices
+
+Consider the following best practices when you construct breadcrumbs:
+
+- Place the breadcrumbs at the top of the page, above the header.
+- Only use breadcrumbs on the second level and deeper of a multi-layered application. Do use breadcrumbs on the first layer of a multi-layered application.
+- Label the breadcrumb to match the page title to which it navigates.
+- Show a maximum of four breadcrumbs. If the application is more than four layers deep, then [truncate the breadcrumb path](#too-many).
+- Show the top hierarchical level as the first breadcrumb. Show the current location as the last breadcrumb. Use a middle breadcrumb to show one level above the current location.
+- If the page title is too long, use a shortened or [truncated]({{site.baseurl}}/style/text-conventions/punctuation.html#Ellipses) version of the page title. For example, instead of `Billing overview`, use `Overview`. Shorter labels conserve horizontal space.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: 1. Can't handle that now. Punt.
2. Worked on wording.
3. When to use - got that.
4. Need Ryan's input. And when breadcrumbs disappear - bring it up in bowling.
5. Keep it for the moment.

[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md [hunk 10]
```diff
@@ -18,30 +18,84 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct breadcrumbs:
+
+- Place the breadcrumbs at the top of the page, above the header.
+- Only use breadcrumbs on the second level and deeper of a multi-layered application. Do use breadcrumbs on the first layer of a multi-layered application.
+- Label the breadcrumb to match the page title to which it navigates.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: #2 and #3 are done.

[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: #4 on bowling agenda.

[the remark this answers is not in the document]
Comment (chri2547) [reply _component_design/breadcrumbs.md]: all questions resolved, done.

**Comment by cyrichardson** [comment]
@chri2547 On it.

**Comment by cyrichardson** [comment]
@chri2547 Some suggested here: 
[Helix_breadcrumbs_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2819239/Helix_breadcrumbs_cyrEdits.docx)

File: _component_design/breadcrumbs.md [hunk 11]
```diff
@@ -18,30 +18,58 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
 
-### When to use
+## Best practices
+
+Consider the following best practices when you construct breadcrumbs:
+
+- Place the breadcrumbs at the top of the page, above the header.
+- Only use breadcrumbs on the second level and lower of a multilayered application. Do use breadcrumbs on the top level of a multilayered application.
+- Before you incorporate breadcrumbs, consider whether you should reserve space on the top-level page so that content does not jump around as the user navigates from page to page.
+- Label the breadcrumb to match the page title to which it navigates. Use title-case capitalization to label the breadcrumbs. For more information about page titles, see [Capitalization]({{site.baseurl}}\style\capitalization.html#title-style-capitalization).
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: is this two separate bullet points?

File: _component_design/breadcrumbs.md [hunk 12]
```diff
@@ -51,65 +79,126 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
...
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Specifications
+### Style
 
-- __Placement:__ Keep breadcrumbs above the major heading of the page, 8px from the eyebrow. The container for breadcrumbs will always live on the top left corner of the page when it is present. Make sure the breadcrumb container is 16px above the content on the top of the page.
-- __Container & Spacing:__ The breadcrumbs container is 28px tall and full-page wide if it needs to be. We recommend shortening measures below. Breadcrumbs are in all caps with arrow icons in `gray 600` in-between.
-- __Alignment:__ Breadcrumbs should be left aligned to the left-most content on the page as noted in [Layout]({{site.baseurl}}/layout/page-layouts.html).
+Use these specifications when you construct breadcrumbs.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: need a link bubble ref to link in the visual

File: _component_design/breadcrumbs.md [hunk 13]
```diff
@@ -51,65 +79,126 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
...
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Too many items
+## States
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: need to add correct hex code for the current page

File: _component_design/breadcrumbs.md [hunk 14]
```diff
@@ -4,11 +4,11 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Breadcrumbs are an inline visual representation of page relationships and hierarchy. They provide a user with visual reinforcement of the current location within the application hierarchy, and allow for quick movement up the hierarchy.
+  Breadcrumbs are a page navigation component that shows the user's current location within a web application. Breadcrumbs help with wayfinding and provide an easy way for the user to navigate to pages located up the application hierarchy.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: "within the hierarchy of a web"

File: _component_design/breadcrumbs.md [hunk 15]
```diff
@@ -18,30 +18,59 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
...
 <section class="static-section" markdown="1">
 
-Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page hierarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second level of nesting and deeper within a module.
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use breadcrumbs when you want to improve web application wayfinding and navigation on sites that organize content hierarchically. For example, breadcrumbs help users better identify where they are. This is particularly true of sites that use top navigation.
```
Comment (palindromebart) [inline _component_design/breadcrumbs.md]: nix "For example, breadcrumbs help users better identify where they are"

File: _component_design/breadcrumbs.md (hunk 11 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 12 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 13 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 14 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

File: _component_design/breadcrumbs.md (hunk 15 above)
Comment (chri2547) [reply _component_design/breadcrumbs.md]: Done.

**Comment by palindromebart** [comment]
Ship it, Squirrel

File: _component_design/breadcrumbs.md (hunk 6 above)
Comment (nimbinatus) [reply _component_design/breadcrumbs.md]: @chri2547 is this discussion resolved?

Comment (chri2547) [reply _component_design/breadcrumbs.md]: Yes, this is resolved. Done.

**Comment by nimbinatus** [comment]
:shipit:

**Pull request merged**